cli/src/modules/message.rs

```diff
@@ -0,0 +1,158 @@
+use kaspa_addresses::Version;
+use kaspa_bip32::secp256k1::XOnlyPublicKey;
+use kaspa_wallet_core::message::{sign_message, verify_message, PersonalMessage};
+
+use crate::imports::*;
+
+#[derive(Default)]
+pub struct Message;
+
+#[async_trait]
+impl Handler for Message {
+    fn verb(&self, _ctx: &Arc<dyn Context>) -> Option<&'static str> {
+        Some("message")
+    }
+
+    fn help(&self, _ctx: &Arc<dyn Context>) -> &'static str {
+        "Sign a message or verify a message signature"
+    }
+
+    async fn handle(self: Arc<Self>, ctx: &Arc<dyn Context>, argv: Vec<String>, cmd: &str) -> cli::Result<()> {
+        let ctx = ctx.clone().downcast_arc::<KaspaCli>()?;
+        self.main(ctx, argv, cmd).await.map_err(|e| e.into())
+    }
+}
+
+impl Message {
+    async fn main(self: Arc<Self>, ctx: Arc<KaspaCli>, argv: Vec<String>, _cmd: &str) -> Result<()> {
+        if argv.is_empty() {
+            return self.display_help(ctx, argv).await;
+        }
+
+        match argv.get(0).unwrap().as_str() {
+            "sign" => {
+                if argv.len() < 2 {
+                    return self.display_help(ctx, argv).await;
+                }
+
+                let kaspa_address = argv.get(1).unwrap().as_str();
```

Comment:
If there's no argument, you can try to get current active account or ask to select it